website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -547,6 +547,46 @@ In this case the table layout is still set to `auto` (default). If instead you w
 
 ### More examples
 
+#### Visually hidden table headers
+
+Labels within the table header are intended to provide contextual information about the column's content to the end user. There may be special cases in which that label is redundant from a visual perspective, because the kind of content can be inferred by looking at it (eg. a contextual dropdown). In this case it's possible to visually hide the label by passing `isVisuallyHidden=true` to the column (only for non-sortable tables).
+
```

Comment:
For clarity, I would call out which column has the header hidden (ie "the last column header" / etc) When looking at the example I was initially confused because I was expecting an example with all the table headers to be hidden (which may not be a valid use case but that was my initial expectation).

Also, if we don't want users to hide all table headers then I think it would be helpful to mention that they should not use isVisuallyHidden to do that.

Reply:
I think the chance that consumers will want to have a table with all the headers hidden would be very remote. It would be more likely that they would try to have a table completely without the table headers, which is something that already happened (and we provided guidance on why that was not OK). So I would not worry too much about this, but good thinking anyway 👍